src/ring_theory/polynomial/chebyshev/basic.lean

```diff
@@ -17,13 +18,13 @@ The Chebyshev polynomials are two families of polynomials indexed by `ℕ`,
 with integral coefficients.
 In this file, we only consider Chebyshev polynomials of the first kind.
 
-## Main declarations
+## Main statements
 
 * `polynomial.chebyshev₁_mul`, the `(m * n)`-th Chebyshev polynomial is the composition
   of the `m`-th and `n`-th Chebyshev polynomials.
-* `polynomial.lambdashev_mul`, the `(m * n)`-th lambdashev polynomial is the composition
+* `polynomial.dickson_one_one_mul`, the `(m * n)`-th lambdashev polynomial is the composition
```

Comment:
Presumably you want to replace "lambdashev" here and in the next few lines with "Dickson polynomial of the first kind"?